oauthenticator/oauth2.py

```diff
@@ -94,6 +94,11 @@ def _client_id_default(self):
     def _client_secret_default(self):
         return os.getenv(self.client_secret_env, '')
 
+    validate_server_cert_env = 'VALIDATE_SERVER_CERT'
+    validate_server_cert = Bool(config=True)
+    def _validate_server_cert_default(self):
+        return os.getenv(self.validate_server_cert_env, True)
```

Comment:
Since this is a bool, we'll want to treat '0' as False, other set values as True, and the default ('' or unset) as True:

```python
env_value = os.getenv(self.validate_server_cert_env, '')
if env_value == '0':
    return False
else:
    return True
```